lib/diffit/timestamp.rb

```diff
@@ -0,0 +1,25 @@
+# encoding: utf-8
+
+module Diffit
+  # Provides a value converter to datetime
+  #
+  module Timestamp
```

Comment:
Decoupling of timestamp creation from a tracker. IMO this conversion messes the tracker's initializer.
It would be even better to provide special timestamp-ish value object, initialized with any value responding to `to_i` or `to_timestamp`, but this utility module also does the work.